### ISSUE TITLE
Failed Install Ubuntu 14.04 (Packaging error)

### ISSUE THREAD

**Issue by Lucif3r** [mail-in-a-box/mailinabox#1086]
root@mail:~# curl -s https://mailinabox.email/setup.sh | sudo bash
FAILED: pip3 install email_validator>=1.0.0
-----------------------------------------
Traceback (most recent call last):
  File "/usr/bin/pip3", line 5, in <module>
    from pkg_resources import load_entry_point
  File "/usr/local/lib/python3.4/dist-packages/pkg_resources/__init__.py", line 70, in <module>
    import packaging.version
ImportError: No module named 'packaging'
-----------------------------------------

**Comment by show0k** [comment]
Duplicate of https://github.com/mail-in-a-box/mailinabox/issues/1080

**Comment by cromulus** [comment]
fixed in #1083, waiting for testing and merge.

In the meantime, *I think* you can address this by running `python3 -m pip install setuptools` and then the installation script. Please report back.

**Comment by axd** [comment]
@cromulus I've seen that you've responded the same to everyone, the problem here is that you can't run any command using pip, for some reason is broken, you have to reinstall python-request as I mentioned here: https://github.com/mail-in-a-box/mailinabox/issues/1080#issuecomment-275160143 I've tried your method and pip always return that error message, so is not a problem of setuptools only.

If you can check your fix commit and add this fix (install or re-install python3-request) would be nice.

Cheers.

**Comment by cromulus** [comment]
The commit won't fix a broken system. The commit only enables new installations to work.

I'll spin up a new vm to see if I can figure out a way to fix a broken system. The suggestion above came from the maintainers of pip itself, and they weren't certain it would fix things.

The challenge is that the problem is a race condition, so it's unclear what state your particular machine is in. `python3 -m pip install setuptools` *could* fix your machine, or not. I just don't know.

**Comment by cromulus** [comment]
ok, I've fixed a broken box with this: 
```
python3 -m pip install --upgrade setuptools 
python3 -m pip install --upgrade distribute
```

Once I get home, I'll again to ensure that this is the right fix.

**Comment by cromulus** [comment]
Yup. tested it on another clean VM.

**Comment by JoshData** [comment]
Fixed in v0.21c posted this morning.
